streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -223,7 +223,7 @@ private void validateStoreOpen() {
         try {
             return this.db.get(rawKey);
         } catch (final RocksDBException e) {
-            throw new ProcessorStateException("Error while getting value for key from store " + this.name, e);
+            throw new ProcessorStateException("Error while getting value for key %s from store " + this.name, e);
```

Comment:
Need to parse this twice before I understood this. I think, we need to add a comment here -- otherwise, we might remove `%s` later as no formatter is used here and thus it's unclear why the markup is there.